lib/src/display/graphic/PicoJpeg.cpp

```diff
@@ -281,35 +281,35 @@ namespace stm32plus {
         case 0:
           return 0;
         case 1:
-          return ((-1) << 1) + 1;
+          return (int16_t)(0x7FFF << 1) + 1;
```

Comment:
Are you sure this was intended?
For all other lines you use `0xFFFF`, but here you use `0x7FFF`